tests/integration/update_cluster/additionalobjects/kubernetes.tf

```diff
@@ -636,7 +646,7 @@ resource "aws_s3_object" "manifests-static-kube-apiserver-healthcheck" {
 resource "aws_s3_object" "nodeupconfig-master-us-test-1a" {
   bucket                 = "testingBucket"
   content                = file("${path.module}/data/aws_s3_object_nodeupconfig-master-us-test-1a_content")
-  key                    = "tests/additionalobjects.example.com/igconfig/master/master-us-test-1a/nodeupconfig.yaml"
+  key                    = "tests/additionalobjects.example.com/igconfig/control-plane/master-us-test-1a/nodeupconfig.yaml"
```

Comment:
will using `--target direct` cleanup the object at the old location or will it be orphaned? and if orphaned, will `kops delete cluster` delete the orphaned object or fail because it doesn't "recognize" the object key?

Reply:
I ran a test. They are orphaned. `kops delete cluster` deletes the orphaned objects anyway.